src/app/Core/ExportServices/generateFilteredDataString.service.ts

```diff
@@ -6,14 +6,14 @@ import { generateCSVString } from "./generateCSVString.service";
  * @param {Record<string, string>[]} rawData - dataset
  * @returns {string} - dataset as string
  */
-export function RAWDataStringGenerator(
+export function generateFilteredDataString(

```

Comment:
not so good: now you renamed the universally usable function so it suggest to be only work with filteredData, but the name is better than the previous one